src/scripts/main.js

```diff
@@ -1,3 +1,33 @@
 'use strict';
 
-// write code here
+let employees = Array.from(document.querySelectorAll('ul li'));
+
+function sortList(list) {
+  employees = list.sort((a, b) => {
+    const salaryA = parseFloat(
+      a.getAttribute('data-salary').replace(/[^\d.-]/g, ''),
+    );
+    const salaryB = parseFloat(
+      b.getAttribute('data-salary').replace(/[^\d.-]/g, ''),
+    );
+
+    return salaryB - salaryA;
+  });
+
+  return employees;
+}
+
+function getEmployees(list) {
+  const employeeList = document.querySelector('ul');
+
+  employeeList.innerHTML = '';
+
+  list.forEach((employee) => {
+    employeeList.appendChild(employee);
+  });
+
+  return list;
+}
+
+sortList(employees);
```

Comment:
The `sortList` function is called without passing any arguments. This might not be the intended behavior since the function expects a list parameter. Consider passing the `employees` list as an argument to the `sortList` function.